packages/renderer/src/lib/ui/Typeahead.spec.ts

```diff
@@ -143,6 +143,31 @@ test('should list items started with search term on top', async () => {
   });
 });
 
+test('should list items started with docker.io + search term on top', async () => {
+  const searchFunction = async () => {
+    await new Promise(resolve => setTimeout(resolve, 100));
+    return ['docker.io/aimage', 'docker.io/bimage', 'docker.io/cimage'];
+  };
+  render(Typeahead, {
+    initialFocus: true,
+    searchFunction,
+    delay: 10,
+  });
+
+  const input = screen.getByRole('textbox');
+
+  await userEvent.type(input, 'cimage');
+
+  await waitFor(() => {
+    const list = screen.getByRole('row');
+    const items = within(list).getAllByRole('button');
+    expect(items.length).toBe(3);
+    expect(items[0].textContent).toBe('docker.io/cimage');
```

Comment:
Why is this listed first ?

Reply:
Because by default if `cimage` used to pull image `docker.io/cimage` is pulled. In search when sorting out proposals for pull, when you type `cimage` the search suggests `docker.io/cimage` among `docker.io/aimage` and `docker.io/bimage`. Changed comparator will put `docker.io/cimage` first then sort everything else in using string.localCompare() function.